src/course-unit/constants.js

```diff
@@ -44,3 +45,26 @@ export const COMPONENT_TYPE_ICON_MAP = {
   [COMPONENT_ICON_TYPES.video]: VideoCameraIcon,
   [COMPONENT_ICON_TYPES.dragAndDrop]: BackHandIcon,
 };
+
+export const getUnitReleaseStatus = (intl) => ({
+  release: intl.formatMessage(messages.releaseStatusTitle),
+  released: intl.formatMessage(messages.releasedStatusTitle),
+  scheduled: intl.formatMessage(messages.scheduledStatusTitle),
+});
+
+export const UNIT_VISIBILITY_STATES = {
+  staffOnly: 'staff_only',
+  live: 'live',
+  ready: 'ready',
+};
+
+export const COLORS = {
+  BLACK: '#000',
+  GREEN: '#0D7D4D',
+};
```

Comment:
Can we make the constant name more indicative of what it's going to be used for?  Like `ICON_COLOR_VARIANTS`, for instance.

(Food for thought: if there was a simple way for us to make these colors semantic and themable instead of hard-coded, it would be awesome.   I'm not going to block the PR on this, though.)

Reply:
Initially, I assumed that these colors could become a “universal color palette from the Javascript side”. But now I think your suggestion is more appropriate because we are using these colors exclusively for the sidebar icon. 
Thank you 💯